esp/esp/program/modules/handlers/teacheracknowledgementmodule.py

```diff
@@ -49,7 +52,7 @@ def acknowledgement(self, request, tl, one, two, module, extra, prog):
             else:
                 rec.delete()
         elif self.isCompleted():
-            context['form'] = teacheracknowledgementform_factory(prog)({'acknowledgement': True})
+            context['form'] = teacheracknowledgementform_factory(prog)(dict([('acknowledgement_%s'%i, True) for i in range(3)]))
```

Comment:
As Ted said above, we don't even use this module anymore, so feel free to delete it.